ansible/roles/docker/files/cc-selector.sh

```diff
@@ -44,10 +47,16 @@ function run {
 
   export ARCH="${arm_type}l"
   export DESTCPU=arm
-  export CC_host="ccache gcc-${gcc_host_version} -m32"
-  export CXX_host="ccache g++-${gcc_host_version} -m32"
+  if [ "$host_os" = "rhel8" ]; then
+    export CC_host="ccache gcc -m32"
```

Comment:
you're sure this is going to be OK? do we not have the option to version this? IIRC we added the host version because it was picking the wrong version because it happened to be the default on the machine, so we went with the version specifier. But maybe that's not an option with rhel in this setup?

Reply:
The RHEL 8 option would be GCC toolsets (the equivalent of devtoolsets in CentOS/RHEL 7). In those cases it's the path to the toolset that changes, not the name of the binary. I could add a check if the gcc version is not 8 (the default in RHEL 8) and attempt to load a GCC toolset, although the container hasn't installed any extra ones.

Refs: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/developing_c_and_cpp_applications_in_rhel_8/additional-toolsets-for-development_developing-applications

Reply:
I've updated this to attempt to load a gcc-toolset if the host gcc version doesn't match.